chrome_extension/cardboard/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "manifest_version": 3,
+  "name": "Cardboard Chrome Extension",
+  "version": "1.0",
+  "icons": {
+    "16": "images/cardboard_bird.png",
+    "32": "images/cardboard_bird.png",
+    "48": "images/cardboard_bird.png",
+    "128": "images/cardboard_bird.png"
+  },
+  "action": {
+    "default_popup": "popup.html"
+  },
+  "permissions": [
+    "activeTab",
+	"cookies"
+  ],
+  "host_permissions": [

```

Comment:
I think also 0.0.0.0:8000, at least that's the URL i use with docker

Reply:
Added